### PR TITLE
Clicking header renders launch page

### PR DESCRIPTION
**Pull request by chandrashekar27**
Made the changes specified in the issue. Now clicking the header (GSoC Analyser) rerenders the launch page. #37

### CODE REVIEW

**Review by Sparsh1212 — CHANGES_REQUESTED** [review]
A good start, but I have a few concerns. The current changes do not clear the input properly. 
What seems a better approach to me is to create an initialState object in the `AdvancedSearch` component. Create a `resetSearchState` function in the `AdvancedSearch` component which sets all the states to the initial state object. That will ensure that all the values are being reset properly. Also note that you will have to invoke the function from the parent component, i.e the `Home` component. There are pretty good answers on StackOverflow regarding invoking a child component function from a parent. Please refer to that if needed.

This shall be the body of `reRenderLauncher`:
```
const reRenderLauncher = () => {
    resetSearchState();
    setValidList([]);
    setDisplayLauncher(true);
}
```

**Comment by Sparsh1212** [comment]
> A good start, but I have a few concerns. The current changes do not clear the input properly. What seems a better approach to me is to create an initialState object in the `AdvancedSearch` component. Create a `resetSearchState` function in the `AdvancedSearch` component which sets all the states to the initial state object. That will ensure that all the values are being reset properly. Also note that you will have to invoke the function from the parent component, i.e the `Home` component. There are pretty good answers on StackOverflow regarding invoking a child component function from a parent. Please refer to that if needed.
> 
> This shall be the body of `reRenderLauncher`:
> 
> ```
> const reRenderLauncher = () => {
>     resetSearchState();
>     setValidList([]);
>     setDisplayLauncher(true);
> }
> ```

**Comment by chandrashekar27** [comment]
@Sparsh1212 
Alright, will work on it.

**Comment by chandrashekar27** [comment]
Hey @Sparsh1212 ! 
I made the changes you requested and it works!! Please take a look.

Also, I had a little trouble with merging the changes made by others(into your `master` branch) into my `issue#37` branch.
It would be great if you could explain the correct method to do that.

**Review by Sparsh1212 — CHANGES_REQUESTED** [review]
Looks really nice to me.
Just one suggestion:
Please make only the Header text clickable. Not the entire Header space. Currently, if I even click outside the header text, it is getting re-rendered.

File: src/components/AdvancedSearch.js [hunk 1]
```diff
@@ -40,6 +43,17 @@ const AdvancedSearch = props => {
   const [suggestions, setSuggestions] = useState([])
   const [isInputInFocus, setIsInputInFocus] = useState(false)
 
+  useImperativeHandle(ref, () => ({
+    resetSearchState () {
+      console.log("YAYYYY!!!!!!");
```
Comment (Sparsh1212) [inline src/components/AdvancedSearch.js:48]: A nice success indicator,
But we need to remove it :)

**Comment by Sparsh1212** [comment]
> Hey @Sparsh1212 ! I made the changes you requested and it works!! Please take a look.
> 
> Also, I had a little trouble with merging the changes made by others(into your `master` branch) into my `issue#37` branch. It would be great if you could explain the correct method to do that.

Ahh, I see. This is because you are facing merge conflicts. I'd suggest reading or watching some tutorials on resolving merge conflicts. Let me know if you are not able to find a relevant resource.

File: src/components/AdvancedSearch.js (hunk 1 above)
Comment (chandrashekar27) [reply src/components/AdvancedSearch.js:48]: Ohh... I thought I had deleted that!

**Comment by chandrashekar27** [comment]
Ok, will make the changes.

**Comment by chandrashekar27** [comment]
Have made changes. Please review @Sparsh1212

**Comment by Sparsh1212** [comment]
LGTM!

**Review by Sparsh1212 — APPROVED** [review]
Nice work :)

**Pull request merged**